### PR TITLE
[stable/openvpn] adding boolean property, which allows to skip default route

### LINKED ISSUE #20265: [stable/openvpn] default route should be optional
**Is your feature request related to a problem? Please describe.**
stable/openvpn chart pushes default route, which is calculates basing on routes inside its own pod.
This default route breaks connectivity for us, because we have kubernetes pods and nodes in the same subnet.

**Describe the solution you'd like**
Add a boolean property in the chart, to configure whether to set default route or not. 

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by irozet**
#### What this PR does / why we need it:
PR adds boolean values to stable/openvpn chart, which allows to skip setting the default route
#### Which issue this PR fixes
fixes #20265 

#### Special notes for your reviewer:
Default route which openvpn chart sets, breaks connectivity in case when pods and nodes are in the same subnet.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/openvpn/README.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ Parameter | Description | Default
 `openvpn.OVPN_K8S_POD_SUBNET`        | Kubernetes pod network subnet (optional)                             | `255.0.0.0`
 `openvpn.OVPN_K8S_SVC_NETWORK`       | Kubernetes service network (optional)                                | `nil`
 `openvpn.OVPN_K8S_SVC_SUBNET`        | Kubernetes service network subnet (optional)                         | `nil`
+`openvpn.SKIP_DEFAULT_ROUTE`         | Do not push a route which openvpn sets by default (optional)         | `nil`
```
Comment (jasongwartz) [inline stable/openvpn/README.md]: can we make this `DEFAULT_ROUTE_ENABLED` and set the default value to `true`?

**Review by jasongwartz — COMMENTED** [review]
thanks for the contribution

\ok-to-test

**Comment by jasongwartz** [comment]
oops, got the slash backwards

/ok-to-test

File: stable/openvpn/README.md (hunk 1 above)
Comment (irozet) [reply stable/openvpn/README.md]: fixed

**Review by jasongwartz — APPROVED** [review]
/lgtm

**Pull request merged**